pkgs/by-name/gi/gibo/package.nix

```diff
@@ -1,40 +1,50 @@
 {
   lib,
-  stdenv,
+  buildGoModule,
   fetchFromGitHub,
-  coreutils,
-  findutils,
-  git,
+  installShellFiles,
 }:
-
-stdenv.mkDerivation rec {
+buildGoModule (finalAttrs: {
   pname = "gibo";
-  version = "1.0.6";
+  version = "3.0.14";
 
   src = fetchFromGitHub {
     owner = "simonwhitaker";
     repo = "gibo";
-    rev = version;
-    sha256 = "07j3sv9ar9l074krajw8nfmsfmdp836irsbd053dbqk2v880gfm6";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-6w+qhwOHkfKt0hgKO98L6Si0RNJN+CXOOFzGlvxFjcA=";
   };
 
-  installPhase = ''
-    mkdir -p $out/bin $out/share/bash-completion/completions
-    cp gibo $out/bin
-    cp gibo-completion.bash $out/share/bash-completion/completions
+  vendorHash = "sha256-pD+7yvBydg1+BQFP0G8rRYTCO//Wg/6pzY19DLs42Gk=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/simonwhitaker/gibo/cmd.version=v${finalAttrs.version}"
+  ];
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    installShellCompletion --cmd gibo \
+      --bash <($out/bin/gibo completion bash) \
+      --fish <($out/bin/gibo completion fish) \
+      --zsh <($out/bin/gibo completion zsh)
+  '';
 
-    sed -e 's|\<git |${git}/bin/git |g' \
-        -e 's|\<basename |${coreutils}/bin/basename |g' \
-        -i "$out/bin/gibo"
-    sed -e 's|\<find |${findutils}/bin/find |g' \
-        -i "$out/share/bash-completion/completions/gibo-completion.bash"
+  installCheckPhase = ''
+    runHook preInstallCheck
+    $out/bin/gibo version | grep -F "${finalAttrs.version}"
+    runHook postInstallCheck
   '';
 
   meta = {
     homepage = "https://github.com/simonwhitaker/gibo";
-    license = lib.licenses.publicDomain;
+    license = lib.licenses.unlicense;
```

Comment:
This seems to have been a mistake in the initial package creation, or the (very early) [change to unlicense](https://github.com/simonwhitaker/gibo/commit/6e69c15c811b0508979582d840aec4ed9abb39f8) was missed. But, please link to the actual license (https://github.com/simonwhitaker/gibo/blob/main/LICENSE) in your commit message changing it, makes researching/blaming this change later simpler.

Reply:
The commit got  with `git log --grep license` doesn't seem to have any URLs, so I ask, but is the URL to be saved the one to `blob/main` in the commit for the license change?

Reply:
I've added the permalink, too.